assembl/templates/base.jinja2

```diff
@@ -6,20 +6,46 @@
     <title>{% block page_title %}Assembl{% endblock %}</title>
     <meta name="viewport" content="width=device-width, initial-scale = 1,user-scalable=no,maximum-scale=1.0">
 
-    {% block meta_tags %}
-    <meta name="description" content="">
-    <meta property="og:title" content="Assembl" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="{{ application_url }}" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/crowd2.jpg" />
-    <meta property="og:image:width" content="1600" />
-    <meta property="og:image:height" content="222" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/logo.png" />
-    <meta property="og:image:width" content="111" />
-    <meta property="og:image:height" content="35" />
-    <meta name="robots" content="noindex" />
-    <meta name="googlebot" content="noindex" />
-    {% endblock meta_tags %}
+    {% block meta_tags_public %}
+        {% if not private_social_sharing %}
+            <meta name="description" content="{{get_description}}">
+            <meta property="og:title" content="{{get_topic}}" />
+            <meta property="og:type" content="website" />
+            <meta property="og:url" content="{{ application_url }}" />
+            <meta property="og:image" content="{{get_landing_page_image}}" />
+            <meta property="og:image:width" content="1600" />
```

Comment:
Have you tried to use real image width and height? Use them if possible, otherwise is it allowed by the standard to just not provide this information?

Reply:
Well on the twitter documentation, this was the given dimension and it appears fine when tested, I don't really get what you mean by real image width and height :D